documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -512,6 +515,14 @@ Start with the default configuration and see if that fits you.
 You can fine tune these setting when you are closer to going in production.
 ====
 
+====
+
+In order to enable `Clustered Counter` support we have defined a `counters` tag.
+Cluster counter support it is used to implement a reliable and consist stategy to generate unique source ID value.
+Cluster counter configuration require jGroups tranport defined configuration.
```

Comment:
s/require/requires/
s/jGroups/JGroups/

Not sure of what you want to say here. Maybe "requires the definition of a JGroups tranport configuration." would be better?

HTH